templates/taxi/car_detail.html

```diff
@@ -1,8 +1,17 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <ul class="float-right btn-group">
+    <a class="btn btn-secondary" href="{% url "taxi:car-update" car.id %}">Edit</a>
```

Comment:
Consider using single quotes in the `{% url %}` template tag: `{% url 'taxi:car-update' car.id %}`. This is a recommended practice to avoid conflicts with HTML attributes that use double quotes.